ui/app/src/components/PathNavigator/PathNavigatorUI.tsx

```diff
@@ -261,6 +260,16 @@ export function PathNavigatorUI(props: PathNavigatorUIProps) {
             fallback: <NavLoader numOfItems={state.itemsInPath?.length > 0 ? state.itemsInPath.length : state.limit} />
           }}
         >
+          {state.currentPath === state.rootPath && (
+            <NavItem
+              item={itemsByPath[state.rootPath]}
+              locale={state.localeCode}
+              isLevelDescriptor={false}
+              onOpenItemMenu={onCurrentParentMenu}
+              onItemClicked={onItemClicked}
+              isRoot={true}
+            />
+          )}
```

Comment:
Every path & every page should show parent item. Not just the root path.
```suggestion

            <NavItem
              item={itemsByPath[state.rootPath]}
              locale={state.localeCode}
              isLevelDescriptor={false}
              onOpenItemMenu={onCurrentParentMenu}
              onItemClicked={onItemClicked}
              isRoot
            />

```